base_layer/core/src/blocks/genesis_block.rs

```diff
@@ -219,29 +219,29 @@ fn get_dibbler_genesis_block_raw() -> Block {
     );
     let mut body = AggregateBody::new(
         vec![],
-        vec![TransactionOutput {
-            features: OutputFeatures {
+        vec![TransactionOutput::new_current_version(
```

Comment:
You have changed the hash of the output, you need to update the MMR in the header of the genesis block as well.